source/agora/node/Registry.d

```diff
@@ -253,8 +254,9 @@ public class NameRegistry: NameRegistryAPI
     {
         TYPE payload_type = this.ensureValidPayload(registry_payload, this.flash.map);
 
-        ensure(isValidChannelOpen(channel.conf, this.agora_node.getBlock(channel.height)),
-            "Not a valid channel");
+        auto range = this.ledger.getBlocksFrom(channel.height);
```

Comment:
Previously it was returning `HTTP:400` when `channel.height` is wrong, it will result `HTTP:500` now.